src/pages/proposals.astro

```diff
@@ -72,4 +74,5 @@ import VoteList from '../components/ProposalsPagenatedList.tsx';
       ]}
     />
   </main>
+<ProposeForm client:load/>
```

Comment:
QUIBBLE: do we want this inside the `<main/>` element, or outside? I can't think of why it would matter, unless it makes a difference for accessibility.

Reply:
You know what, answered my own question. I think what you did makes the most sense 👍